docs/framework-decorator-client.md

```diff
@@ -215,6 +215,36 @@ delete_user(1, query={"hard": "false"})
     - Collections and other annotated types use `TypeAdapter` (Pydantic v2) or `parse_obj_as` (Pydantic v1).
     - If no return annotation is present, the raw payload is returned without validation.
 
+## Persistent connections with context managers
```

Comment:
Let's drop all the context manager stuff - both the documentation and the code for it, as I feel like it is not a necessary feature.